notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb

```diff
@@ -482,14 +441,14 @@
         "id": "container:prediction"
       },
       "source": [
-        "### Set pre-built containers\n",
+        "### Set prebuilt containers\n",
         "\n",
-        "Set the pre-built Docker container image for prediction.\n",
+        "Set the prebuilt Docker container image for prediction.\n",
         "\n",
-        "- Set the variable `TF` to the TensorFlow version of the container image. For example, `2-1` would be version 2.1, and `1-15` would be version 1.15. The following list shows some of the pre-built images available:\n",
+        "- Set the variable `TF` to the TensorFlow version of the container image. The following list shows some of the prebuilt images available:\n",
         "\n",
         "\n",
-        "For the latest list, see [Pre-built containers for prediction](https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers)."
+        "For the latest list, see [prebuilt containers for prediction](https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers)."
```

Comment:
- update this URL:  https://cloud.google.com/ai-platform-unified/docs/predictions/pre-built-containers
- to:                          https://cloud.google.com/vertex-ai/docs/predictions/pre-built-containers

Avoids a redirect